postgres.go

```diff
@@ -35,33 +35,42 @@ const (
 
 // A PostgresDatabase is an instance of Linode Postgres Managed Databases
 type PostgresDatabase struct {
-	ID                    int                       `json:"id"`
-	Status                DatabaseStatus            `json:"status"`
-	Label                 string                    `json:"label"`
-	Region                string                    `json:"region"`
-	Type                  string                    `json:"type"`
-	Engine                string                    `json:"engine"`
-	Version               string                    `json:"version"`
-	Encrypted             bool                      `json:"encrypted"`
-	AllowList             []string                  `json:"allow_list"`
-	Port                  int                       `json:"port"`
-	SSLConnection         bool                      `json:"ssl_connection"`
-	ClusterSize           int                       `json:"cluster_size"`
-	ReplicationCommitType PostgresCommitType        `json:"replication_commit_type"`
-	ReplicationType       PostgresReplicationType   `json:"replication_type"`
-	Hosts                 DatabaseHost              `json:"hosts"`
-	Updates               DatabaseMaintenanceWindow `json:"updates"`
-	Created               *time.Time                `json:"-"`
-	Updated               *time.Time                `json:"-"`
+	ID            int              `json:"id"`
+	Status        DatabaseStatus   `json:"status"`
+	Label         string           `json:"label"`
+	Region        string           `json:"region"`
+	Type          string           `json:"type"`
+	Engine        string           `json:"engine"`
+	Version       string           `json:"version"`
+	Encrypted     bool             `json:"encrypted"`
+	AllowList     []string         `json:"allow_list"`
+	Port          int              `json:"port"`
+	SSLConnection bool             `json:"ssl_connection"`
+	ClusterSize   int              `json:"cluster_size"`
+	Platform      DatabasePlatform `json:"platform"`
+
+	// Members has dynamic keys so it is a map
+	Members map[string]DatabaseMemberType `json:"members"`
+
+	ReplicationCommitType PostgresCommitType      `json:"replication_commit_type,omitempty"` // This field doesn't exist in DBaaS v2
+	ReplicationType       PostgresReplicationType `json:"replication_type,omitempty"`        // This field doesn't exist in DBaaS v2
+
+	Hosts             DatabaseHost              `json:"hosts"`
+	Updates           DatabaseMaintenanceWindow `json:"updates"`
+	Created           *time.Time                `json:"-"`
+	Updated           *time.Time                `json:"-"`
+	Fork              DatabaseFork              `json:"fork"`
```

Comment:
Looks like fork is null when the database isn't a fork of another database. Could this be made a pointer accordingly?

Reply:
Fixed!